src/mruby-widget-lib/src/gem.c

```diff
@@ -990,19 +1036,43 @@ mrb_remote_param_set_value(mrb_state *mrb, mrb_value self)
             next = (param->max-param->min)*value + param->min;
         br_set_value_int(param->br, param->uri, next);
     } else if(param->type == 'f') {
-        const float x = value;
         float out = 0;
-		if(param->scale && strstr(param->scale, "log")) {
```

Comment:
nothing negative about the changes, just odd that this code used to be indented with tabs.